package.json

```diff
@@ -27,7 +27,7 @@
     "postinstall": "manypkg check && preconstruct dev",
     "build": "preconstruct build",
     "fix": "manypkg fix",
-    "typecheck": "tsc",
+    "typecheck": "tsc --noEmit",
```

Comment:
we already use this setting:
https://github.com/statelyai/xstate/blob/de6591fa6a589cfdc441daf9124cf73086184ef5/tsconfig.json#L8